gammapy/spectrum/tests/test_spectrum_extraction.py

```diff
@@ -43,18 +46,17 @@ def test_spectrum_extraction(pars, results, tmpdir):
         "datasets/exclusion_masks/tevcat_exclusion.fits")
     excl = ExclusionMask.read(exclusion_file)
 
-    irad = Angle('0.5 deg')
-    orad = Angle('0.6 deg')
-    bk = [ring_background_estimate(
-        center, radius, irad, orad, _.events) for _ in obs]
-    # bk = dict(method='reflected', n_min=2, exclusion=excl)
+    bk = dict(method='reflected', n_min=2, exclusion=excl)
 
-    bounds = EnergyBounds.equal_log_spacing(1, 10, 40, unit='TeV')
-    etrue = EnergyBounds.equal_log_spacing(0.1, 30, 100, unit='TeV')
+    # Restrict energy binning to a range where the aeff interpolation does not
+    # give none for HAP test files
+    # TODO: set low energies to 0 and extrapolate high energies
+    e_true = np.logspace(-1, 1.9, 70) * u.TeV
```

Comment:
@registerrier I choose this soluting, e.g. setting the true energy range for the test withing the range where our IRFs are defined. The general solution with extrapolation at high energies and setting to 0 at lower is a little bit tricky to implement in a general way I think. so I skipped it for now